deployments/scripts/deploy-osmo-minimal.sh

```
@@ -247,6 +258,8 @@ while [[ $# -gt 0 ]]; do
             ;;
         --ngc-api-key)
             NGC_API_KEY="$2"; shift 2 ;;
+        --ngc-secret-name)
+            NGC_SECRET_NAME="$2"; shift 2 ;;
```

Shellcheck (0.11.0)

[warning] NGC_SECRET_NAME appears unused. Verify use (or export if used externally).

(SC2034)